### PR TITLE
feat(pano): single post page

### PR DESCRIPTION
**Pull request by FurkanGM**
#510 implementation

### CODE REVIEW

File: db/prisma/schema.prisma [hunk 1]
```diff
@@ -99,7 +99,7 @@ model Post {
 model User {
   id            String    @id @default(cuid())
   name          String? // Check if needed
-  username      String?   @unique
+  username      String   @unique
```
Comment (Ketcap) [inline db/prisma/schema.prisma]: username next-auth'da optional olarak verildigi icin oyle eklenmisti. Bu sonradan sorun cikarir optional olarak birakalim PR'in scope'una ait de degil hem

File: apps/gql/loaders/pano.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+const postsLoaderBatchFn = async (
+  clients: Clients,
+  keys: readonly Partial<PanoQueryPostsArgs>[]
+) => {
+  const results: PanoPostsConnection[] = [];
+
+  for (const key of keys) {
+    const { before, after, first, last } = key;
+
+    const result = await findManyCursorConnection(
```
Comment (Ketcap) [inline apps/gql/loaders/pano.ts]: bu satir icin diger implementasyonlara bakar misin. extra paket direkt eklemek ilk secenek olmasin.

Comment (Ketcap) [reply apps/gql/loaders/pano.ts]: Pagination icin fonksiyonlar var su an onlara bir bak https://github.com/kamp-us/monorepo/blob/073b822cf089c9713785e8db2fd69233cbc43a7a/apps/gql/features/relay/pagination.ts#L36

**Review by Ketcap — CHANGES_REQUESTED** [review]
@FurkanGM PR icin tesekkurler 🤌 testlerini ekleyelim eksik kalmasin

**Comment by usirin** [comment]
Closing this as now we implemented all the functionality that exists here with several other pull requests, you can track the remaining progress from #510 

Thank you so much for making the initial work @FurkanGM it literally gave me the initial roadmap, and helped us figure out the problems early on in the process. This is a great example of creating a PR to start a work and let the rest of the team handle the remaining parts. Thank you for being an example to the community!

**Pull request closed without merging**